pkg/vectorize/rpad/rpad.go

```diff
@@ -280,3 +310,30 @@ func rpadUint64(rowCount int, strs *types.Bytes, sizes []uint64, padstrs *types.
 	}
 	return results, resultNsp
 }
+
+func BytesToInt[T constraints.Signed](xs *types.Bytes, rs []T) ([]T, []bool, error) {
```

Comment:
What does this do?  Atoi?

Reply:
Yeah, like this, the typecast.BytesToInt doesn't satisfy my need. I need more info like flags and so on to solve the problem of
typeBytes of rpad. I don't modify the typecast.BytesToInt, because the modified func is used in many places. I'm afraid it cause many other errors.